queue/queue_service.js

```diff
@@ -6,6 +6,76 @@ queueApp.factory("QueueService", function($http){
 			},function(error){
 				return null;
 			});
+		},
+		checkInNow: function(time, id) {
+			return $http({
+				url: "/server/queue_priv.php",
+				method: 'POST',
+				data: 'action=checkIn&time=' + time + '&id=' + id,
+				headers: {
+					'Content-Type': "application/x-www-form-urlencoded"
+				}
+			}).then(function(response){
+				return response.data;
+			},function(error){
+				return null;
+			});
+		},
+		turnInPapers: function(time, id) {
+			return $http({
+				url: "/server/queue_priv.php",
+				method: 'POST',
+				data: 'action=completePaperwork&time=' + time + '&id=' + id,
```

Comment:
On a sidenote, and there's definitely no need to change this. But we are going to be doing that transpiling from ES6 to ES5, so feel free to use ES6 features like string interpolation:
```
`action=completePaperwork&time=${time}&id=${id}`
````
That will do the same thing as the `+` signs, but it's more concise and tends to be easier to read and less verbose.